index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-First</title>
+
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/nav.css">
+  <link rel="stylesheet" href="styles/info.css">
+  <link rel="stylesheet" href="styles/footer.css">
+</head>
+<body>
+  <main>
+    <header id="header">
+      <div class="header-container">
+        <p class="text-gradient">Welcome</p>
```

Comment:
- [ ] For the logo you should use the `a` tag so that it maybe clickable because a logo is always a link